### DISCUSSION TITLE
[Maintainers] Switching from SSE to AVX, for Thorium

### DISCUSSION THREAD

**Discussion by Ra2-IFV** [ScoopInstaller/Main#6283] [Ideas]
Modern CPU supports AVX, SSE3 is pretty old enough. Considering the purpose of Thorium, most users should benefit from this.
I have a repo that has all 3 versions of Thorium for different CPU instruction sets. I plan to replace the current manifest with the AVX one, then open a pull request for the rest (SSE3 and AVX2)

**Comment by Lutra-Fs** [comment]
Hi as I can see the avx2 version is already available in https://github.com/ScoopInstaller/Versions/commits/master/bucket/thorium-avx2.json
I would recommend you to raise a issue in extras bucket to determine if we need to update the thorium to the AVX2 version by default.
Regards.

**Reply by Ra2-IFV** [reply]
Yeah, I need someone to agree with me before I open an issue.
